src/app/lib/fonts.js

```diff
@@ -1,31 +1,6 @@
+/* eslint-disable no-unused-expressions */
 import { injectGlobal } from 'styled-components';
 
-// Note - this file should only include standard constants,
-// agreed to by UX and Engineering
-
-// Colours
-export const C_EBON = '#222222';
-export const C_POSTBOX = '#BB1919';
-export const C_STORM = '#404040';
-export const C_WHITE = '#FFFFFF';
-
-// Font family
-const fontFamilyBase = ', Arial, Helvetica, freesans, sans-serif';
-export const FF_NEWS_SANS_LIT = `ReithSansNewsLight${fontFamilyBase}`;
-export const FF_NEWS_SANS_REG = `ReithSansNewsRegular${fontFamilyBase}`;
-export const FF_NEWS_SANS_MDM = `ReithSansNewsMedium${fontFamilyBase}`;
-export const FF_NEWS_SANS_BLD = `ReithSansNewsBold${fontFamilyBase}`;
-export const FF_NEWS_SERIF_LIT = `ReithSerifNewsLight${fontFamilyBase}`;
-export const FF_NEWS_SERIF_REG = `ReithSerifNewsRegular${fontFamilyBase}`;
-export const FF_NEWS_SERIF_MDM = `ReithSerifNewsMedium${fontFamilyBase}`;
-export const FF_NEWS_SERIF_BLD = `ReithSerifNewsBold${fontFamilyBase}`;
-
-// GEL Spacing
-export const GEL_SPACING = 8;
-export const GEL_SPACING_DBL = GEL_SPACING * 2;
-
-// Start defining Reith fonts.
-/* eslint-disable no-unused-expressions */
 injectGlobal`
```

Comment:
Are you aware that this is disabling Reith?

Reply:
Fixed